grails-app/views/order/_orderAdjustments.gsp

```diff
@@ -17,9 +17,9 @@
             </tr>
             </thead>
             <tbody>
-            <g:each var="orderAdjustment" in="${orderInstance.orderAdjustments?.sort { it.dateCreated }}" status="status">
+            <g:each var="orderAdjustment" in="${orderInstance.orderAdjustments?.sort()}" status="status">
```

Comment:
Are there other places where we display order adjustments?

Reply:
I can see them on `order/print`
```
orderInstance?.orderAdjustments.findAll { !(it.orderItem || it.canceled) }.sort { it.totalAdjustments }.reverse()
```

Reply:
@alannadolny Ok cool. Can you include that in the ticket (@ Manon) with 1) a brief description of what is happening in that code 2) question on whether these need to be sorted as well?